src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -3,7 +3,7 @@
 namespace Illuminate\Foundation\Exceptions;
 
 use Exception;
-use Psr\Log\LoggerInterface;
+use Illuminate\Contracts\Logging\Log;
```

Comment:
Seeing as the exception handler is [resolved out of the container](https://github.com/laravel/framework/blob/5.1/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php#L154), we need to specifically typehint the `Illuminate\Contracts\Logging\Log` interface so that an `Illuminate\Log\Writer` instance is injected instead of just a `Monolog\Logger` instance.